src/driver/verify.cpp

```diff
@@ -43,24 +51,24 @@ void verify_program(const std::string& name,
                     const program& p,
                     const target& t,
                     compile_options options,
+                    precision quantize,
                     const parameter_map& inputs,
                     double tolerance)
 {
     auto x = run_ref(p, inputs);
```

Comment:
The user should be able to pass in a different tolerance value when comparing to fp32 ref. It might vary depending on the model, so having the user specify the tolerance might be the best option.